sip-servlets-client/src/main/java/org/mobicents/javax/servlet/sip/dns/DNSResolver.java

```diff
@@ -60,7 +61,7 @@
 	 * @param uri the uri used to find the corresponding SipURI
 	 * @return the SipURI found through ENUM methods or the uri itself if the uri is already a SipURI without a user=phone param
 	 */
-	SipURI getSipURI(URI uri);
+	SipURI getSipURI(URI uri, SipSession sipSession);
```

Comment:
Because I was misunderstanding about SIP URI parameter and header parameter. I will start coding again.